src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonIgnoreCondition.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Text.Json.Serialization
+{
+    /// <summary>
+    /// Controls how the <see cref="JsonIgnoreAttribute"/> ignores properties on serialization and deserialization.
+    /// </summary>
+    public enum JsonIgnoreCondition
+    {
+        /// <summary>
+        /// Property will always be ignored.
+        /// </summary>
+        Always = 0,
+        /// <summary>
+        /// Property will only be ignored if it is null.
+        /// </summary>
+        WhenNull = 1,
+        /// <summary>
+        /// Property will always be serialized and deserialized, regardless of <see cref="JsonSerializerOptions"/> configuration.
```

Comment:
Clarify with "JsonSerializerOptions.IgnoreNullValues"?

Reply:
Will do. We may have "JsonSerializerOptions.IgnoreDefaultValues" in the near future, so we'll have to add that too.